Tools/CISupport/ews-build/events.py

```diff
@@ -314,9 +315,41 @@ class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
     OPEN_STATES = ('open',)
 
+    @classmethod
+    def file_with_status_sign(cls, info):
+        if info.get('status') in ('added', 'modified'):
+            return '+++ {}'.format(info['filename'])
+        return '--- {}'.format(info['filename'])
+
     def _get_commit_msg(self, repo, sha):
         return ''
 
+    @defer.inlineCallbacks
+    def _get_pr_files(self, repo, number):
+        # Copied from buildbot/www/hooks/github.py to include added/modified/deleted
+        headers = {"User-Agent": "Buildbot"}
+        if self._token:
+            p = Properties()
+            p.master = self.master
+            token = yield p.render(self._token)
```

Comment:
This looks different than https://github.com/buildbot/buildbot/blob/v2.10.5/master/buildbot/www/hooks/github.py , is that intentional?

Reply:
I guess I got that code from my local buildbot package. It's a bit non-sensical when compared to what they're currently using, I would agree. Changing.